game/scripts/vscripts/TutorialGraph/Steps.ts

```diff
@@ -113,6 +113,91 @@ export const moveUnit = (unit: tg.StepArgument<CDOTA_BaseNPC>, moveLocation: tg.
         })
 }
 
+/**
+ * Creates a tutorial step that orders a unit to use an ability.
+ * @param caster Unit that will use the ability.
+ * @param target Target of the ability.
+ * @param abilityName Name of the ability.
+ * @param orderType Type of unit order used for casting the ability with ExecuteOrderFromTable.
+ */
+export const useAbility = (caster: tg.StepArgument<CDOTA_BaseNPC>, target: tg.StepArgument<CDOTA_BaseNPC> | tg.StepArgument<Vector>, abilityName: tg.StepArgument<string>, orderType: UnitOrder) => {
```

Comment:
This could be a normal util function (and used with immediate) instead of a step, since it doesn't actually do anything async.